source/jormungandr/jormungandr/street_network/asgard.py

```diff
@@ -212,10 +212,12 @@ def _is_cycle_lane(path):
 
         # We have one journey and several sections in direct path
         for section in response.journeys[0].sections:
-            path_items = section.street_network.path_items
-            cycle_lane_length = sum((path.length for path in path_items if _is_cycle_lane(path)))
-            # Since path.length are doubles and we want an int32 in the proto
-            section.cycle_lane_length = int(cycle_lane_length)
+            # do not add cycle_lane_length for bss_rent/bss_return & walking sections
```

Comment:
From what I understand, this should only be for walking sections, not BSS, isn't it?

Reply:
So I didn't understand correctly. Your comment was clear and did mention `bss_rent` and `bss_return`, not just `bss`. So my comment is not relevant.